js/main.js

```diff
@@ -1,5 +1,8 @@
 import {createPhotos} from './mockPhotos.js';
 import {renderPhotos} from './renderingPicture.js';
+import { addHandler } from './popupGallery.js';
```

Comment:
Предлагаю изменить название метода, на какое-то такое. Т.е. идея, насколько я понимаю, будет такая: у нас механизм работы галереи вообще независим от фотографий. Единственная связь с ними через данные, которые мы можем "установить" в нашу галерею. 
```suggestion
import { setPhotos} from './popupGallery.js';
```